x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_factory.ts

```diff
@@ -77,19 +77,21 @@ export const createPersistenceRuleTypeFactory: CreatePersistenceRuleTypeFactory
                 const alertUuid = event['kibana.rac.alert.uuid'];
                 const isAlert = alertUuid != null;
                 return {
-                  ...event,
+                  // ...event,
```

Comment:
Temporary workaround to unblock work, will not be merged. Seems to be newly introduced mapping issues related to the security solution rules that are being worked on.